sparse/numba_backend/tests/test_coo.py

```diff
@@ -1926,3 +1926,29 @@ def test_xH_x():
     assert_eq(Ysp.conj().T @ Y, Y.conj().T @ Y)
     assert_eq(Ysp.conj().T @ Ysp, Y.conj().T @ Y)
     assert_eq(Y.conj().T @ Ysp.conj().T, Y.conj().T @ Y.conj().T)
+
+
+@pytest.mark.parametrize("ndim", range(1, 5))
+@pytest.mark.parametrize("repeats", [1, 2, 3])
+def test_repeat(ndim, repeats):
+    rng = np.random.default_rng()
+    shape = tuple(rng.integers(1, 4) for _ in range(ndim))
+    a = rng.integers(1, 10, size=shape)
+    with pytest.raises(TypeError, match="`a` must be a SparseArray"):
+        sparse.repeat(a, repeats=2)
+    sparse_a = COO.from_numpy(a)
+    with pytest.raises(Exception, match="`repeats` must be an integer"):
+        sparse.repeat(sparse_a, repeats=[2, 2, 2])
+    for axis in range(ndim):
```

Comment:
Ahh -- you test `axis` here. A random element of `[*range(-ndim, ndim), None]` should be good.